zulipterminal/version.py

```diff
@@ -1,4 +1,4 @@
-ZT_VERSION = '0.5.1+git'
+ZT_VERSION = '0.5.2'
 
 SUPPORTED_SERVER_VERSIONS = [
     ('2.1', None),
```

Comment:
Let's remove the comment from the next line.

Reply:
Oh yes, I agree - though that's not shown in this review context...

Reply:
That said, there are features of v3.0 that we don't yet support. Perhaps #731 could include a table for version-specific features.